bin/node/src/commands/store.rs

```diff
@@ -119,7 +125,29 @@ impl StoreCommand {
             .as_secs()
             .try_into()
             .expect("timestamp should fit into u32");
-        let genesis_state = GenesisState::new(vec![account_file.account], version, timestamp);
+        Ok(GenesisState::new(vec![account_file.account], version, timestamp))
+    }
+
+    fn bootstrap(
+        data_directory: &Path,
+        accounts_directory: &Path,
+        maybe_genesis_config_path: Option<&PathBuf>,
+    ) -> anyhow::Result<()> {
+        let genesis_state = if let Some(genesis_config_path) = maybe_genesis_config_path {
+            let toml_str = fs_err::read_to_string(genesis_config_path)?;
+            let config = GenesisConfig::read_toml(toml_str.as_str())
+                .context(format!("Read from file: {}", genesis_config_path.display()))?;
+
+            let (state, secrets) = config.into_state()?;
+            // Write the accounts to disk
+            for account in secrets.as_account_files() {
+                let accountpath = accounts_directory.join(DEFAULT_ACCOUNT_PATH);
```

Comment:
this only creates a single file with secrets that each subsequent account overwrites